ha/k8s_setup/ha_setup.py

```diff
@@ -391,6 +404,12 @@ def _add_node_health(self, data_node_ids, server_node_ids, control_node_ids) ->
                                    resource_type=CLUSTER_ELEMENTS.NODE.value,
                                    resource_id=node_id,
                                    specific_info=specific_info)
+        if node_id_list:
+            # Note: if batch put is enabled needs to commit
+            # to push all the local cashed values to consul server
+            if kv_enable_batch_put:
+                Log.debug(f"pushing node health transactions to store")
+                self._confstore.commit()
```

Comment:
where is the batch populated?

Reply:
https://github.com/Seagate/cortx-ha/blob/f16a715a6f8079b93a125852d0c3fac4d918c305/ha/util/consul_kv_store.py#L330
This is commit and which will internally call batch put:
https://github.com/Seagate/cortx-ha/blob/f16a715a6f8079b93a125852d0c3fac4d918c305/ha/util/consul_kv_store.py#L287